packages/bpk-component-slider/src/BpkSlider.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import Slider from 'react-slider';
+import { cssModules } from 'bpk-react-utils';
+import React from 'react';
+
+import STYLES from './bpk-slider.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkSlider = (props) => {
+  const {
+    min,
+    max,
+    value,
+    step,
+    large,
+    minDistance,
+    className,
+    onChange,
+    onAfterChange,
+  } = props;
+
+  const sliderClass = [getClassName('bpk-slider')];
+  const handleClass = [getClassName('bpk-slider-handle')];
+  const barClass = [getClassName('bpk-slider-bar')];
+
+  if (value.length) sliderClass.push(getClassName('bpk-slider--range'));
+  if (className) sliderClass.push(getClassName(className));
+  if (large) handleClass.push(getClassName('bpk-slider-handle--large'));
+
+  const handleActiveClass = getClassName('bpk-slider-handle--active');
+  return (
+    <div className={getClassName('bpk-slider-wrapper')}>
+      <Slider
+        max={max}
+        min={min}
+        step={step}
+        value={value}
+        minDistance={minDistance}
+        withBars
+        className={sliderClass.join(' ')}
+        handleClassName={handleClass.join(' ')}
+        handleActiveClassName={handleActiveClass}
+        barClassName={barClass.join(' ')}
+        onChange={v => onChange(v)}
+        onAfterChange={v => onAfterChange(v)}
+      />
+    </div>
+  );
+};
+
+BpkSlider.propTypes = {
+  min: PropTypes.number.isRequired,
+  max: PropTypes.number.isRequired,
+  minDistance: PropTypes.number,
+  value: PropTypes.oneOfType([PropTypes.array, PropTypes.number]).isRequired,
```

Comment:
`PropTypes.array` should be `PropTypes.arrayOf(PropTypes.number)` right?

Reply:
Yes, I have changed it